pandas/plotting/_core.py

```diff
@@ -1208,6 +1208,16 @@ def hist(self, by=None, bins=10, **kwargs):
             ...     columns = ['one'])
             >>> df['two'] = df['one'] + np.random.randint(1, 7, 6000)
             >>> ax = df.plot.hist(bins=12, alpha=0.5)
+
+        If `by` is defined, a grouped hist plot is generated:
```

Comment:
is context on what by can be (str, list of str?)

Reply:
it is already defined lines above in `Parameters` section, but I added some words to address it, and add one more example to demonstrate different scenarios